app/views/claims/qts_year.html.erb

```diff
@@ -12,8 +12,14 @@
         </span>
 
         <%= errors_tag current_claim, :qts_award_year %>
-        <%= form.select :qts_award_year, options_for_qts_award_year, { include_blank: true }, { class: css_classes_for_select(current_claim, :qts_award_year) } %>
-
+        <div class="govuk-radios">
+          <%= form.collection_radio_buttons :qts_award_year, options_for_qts_award_year, :last, :first do |b| %>
+            <div class="govuk-radios__item">
+              <%= b.radio_button class: "govuk-radios__input" %>
+              <%= b.label class: "govuk-label govuk-radios__label" %>
+            </div>
+          <% end %>
+        </div>
```

Comment:
Does this need the form helper magic that `form.select` had for handling errors?

Reply:
I don't think so because the radios/checkboxes don't get any extra styling in an error state. https://design-system.service.gov.uk/components/error-message/

Reply:
Currently looks like: 
![errors](https://user-images.githubusercontent.com/2804149/61470666-67419900-a979-11e9-85d0-aee2eab993e0.png)

Reply:
Ok, nice!